react-commenting.js

```diff
@@ -58,10 +58,10 @@ export default App
 class GoodRobot extends Component{
   render(){
     return(
-      // 8.
+      // 8. The div below contains the rendered information from this component that will be plugged into the parent component App in it's own respective render. 
       <div>
         <h3>Good Robot</h3>
-        {/* 9. */}
+        {/* 9. This line takes in the information from the state object userInput by the use of props and displays it along with a static sentence */}
```

Comment:
Yup. I would maybe mention where that information comes from, being the parent component that passes this information in.